docs/design/support-torch-elastic.md

```diff
@@ -0,0 +1,208 @@
+# Design to Support Torch Distributed Elastic
+
+The design is to introduce how to support elastic distributed
+training using `torch.distributed.elastic` in DLRover.
+
+## Motivation
+
+With [Torch Distributed Elastic](https://pytorch.org/docs/stable/distributed.elastic.html)
+, PyTorch can react to worker membership changes or failures and
+restart worker processes. To run a job with elastic PyTorch on
+a distributed cluster, an elastic node scheduler is necessary
+to launch or remove nodes. Now, [TorchX-Kubernetes](https://pytorch.org/torchx/latest/)
+supports to launch Pods on the Kubernetes with the min and max number
+of workers. However, the batch size of sync-SGD will vary with
+the number of workers which may result in inconsistency of model accuracy.
+What's more, how many workers can maximize the training performance
```

Comment:
`What's more, it is difficult for users to figure out the number of workers that can maximize ...`